README.md

```diff
@@ -114,6 +116,8 @@ webgriffe_sylius_akeneo:
 
 For each `translatable_property` value handler you have to configure, in `$translationPropertyPath`, the Sylius product (and product variant) translation property path of the property where to store the value of the Akeneo attribute whose code is configured with `$akeneoAttributeCode`.
 
+In addition, you can say if the translatable property should be allowed to be null or not. This is strongly recommended for the name property. In fact, if you do not specify this option and there is a name with a null value you will get a database error due to the NOT NULL constraint on the product_translation and product_variant_translation name property that cannot be handled correctly and that will cause a block for the queue consuming.
```

Comment:
```suggestion
In addition, you can specify if the translatable property should be allowed to be null or not. **_This is strongly recommended for the name property._** In fact, if you do not specify this option and there is a name with a null value you will get a database error due to the NOT NULL constraint on the product_translation and product_variant_translation name property that cannot be handled correctly and that will cause a block for the queue consuming.
```

We should make it more visible with bold and italic?
Maybe we can even add something about the Importer that, before persisting the changes, it will throw if the name or slug on the product are null or the name of the variant is null.